docs/modules/operation/pages/deep-dive/notifications/concepts.adoc

```diff
@@ -1,114 +1,94 @@
 
-Each step in a destination path has an associated delay which defaults to zero seconds. The initial step's delay is called the initial delay, while an escalation step's delay is simply called its delay.
+== Destination paths
+
+A destination path is a named, reusable set of rules for sending notifications.
+Each destination path has a defined initial step and zero or more escalation steps.
+Every step in a destination path has an associated delay, which defaults to zero seconds.
+The initial step's delay is called the initial delay, while an escalation step's delay is simply called its delay.
 
 Each step has one or more targets.
-A target may be a user, a group, an on-call role, or a one-off e-mail address.
+A target may be a user, a Group, an On-Call Role, or a one-off email address.
 
-NOTE: While it may be tempting to use one-off e-mail addresses any time an individual user is to be targeted, it's a good idea to reserve one-off e-mail addresses for special cases.
-If a user changes her e-mail address, for instance, you'll need to update in every destination path where it appears.
-The use of one-off e-mail addresses is meant for situations where a vendor or other external entity is assisting with troubleshooting in the short term.
+NOTE: While it may be tempting to regularly use one-off email addresses to target individual users, it's a good idea to reserve their use for special cases.
+If a user changes their email address, for example, you will need to update it in every Destination Path in which it appears.
+One-off email addresses are meant for situations where a vendor or other external party is assisting with troubleshooting in the short-term.
 
-When a step targets one or more groups, a delay may also be specified for each group.
-The default is zero seconds, in which case all group members are notified simultaneously.
-If a longer delay is set, the group members will be notified in alphabetical order of their usernames.
+When a step targets one or more Groups, a delay may be specified for each Group.
+The default delay is zero seconds; this means all Group members are notified at the same time.
+If a longer delay is set, Group members will be notified in alphabetical order, based on their usernames.
 
-IMPORTANT: Avoid using the same name for a group and a user.
-The destination path configuration does not distinguish between users and groups at the step level, so the behavior is undefined if you have both a user and a group named `admin`.
-It is for this reason that the default administrators group is called `Admin` (with a capital `A`) -- case matters.
+IMPORTANT: Avoid using the same name for a Group and a user.
+Destination path configuration settings do not distinguish between Users and Groups as the step level.
+If you have both a user and a Group named "admin," the behavior will be undefined.
+Because of this, the default administrators group is called "Admin"—case matters.
```

Comment:
```suggestion
Because of this, the default administrators group is called "Admin" — case matters.
```

Reply:
In this case, we do mention in the Style Guide that there should be no spaces around an em dash. Do we want to update this?

Reply:
My bad. There shouldn't be a space between the em dash. It looked odd after the quotation mark. Ignore my original comment.